CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ You can find its changes [documented below](#060---2020-06-01).
 
 ### Docs
 
+- Added documentation for the `druid::Image` widget. ([#1018] by [@covercash2])
```

Comment:
I think just having `Image` without the qualifer is enough, that's what we've been using for types in the changelog.